aya/src/obj/mod.rs

```diff
@@ -977,6 +1137,29 @@ fn parse_version(data: &[u8], endianness: object::Endianness) -> Result<KernelVe
     })
 }
 
+fn get_map_field(btf: &Btf, type_id: u32) -> Result<u32, BtfError> {
```

Comment:
What does this do exactly? It returns `arr.nelems` so it's not obviously clear. Needs a comment and maybe a better name? :P